website/src/components/canvas/Canvas.jsx

```diff
@@ -59,6 +59,7 @@ class Canvas extends React.Component {
         //fetch canvas specific info
         const canvasObj = {};
         canvasObj[RC.CANVAS_NAME] = canvasSnap.child('name').val();
+        canvasObj[RC.CANVAS_ORIENTATION] = canvasSnap.child('orientation').val();
```

Comment:
I would make a check to first see if orientation is valid here. This may fix some problems.